src/status_im/chat/events/input.cljs

```diff
@@ -0,0 +1,496 @@
+
+(register-handler-fx
+ :load-chat-parameter-box
+ [trim-v]
+ (fn [{{:keys [current-chat-id bot-db current-account-id] :as db} :db}
+      [{:keys [name type bot owner-id] :as command}]]
+   (let [parameter-index (input-model/argument-position db current-chat-id)]
+     (when (and command (> parameter-index -1))
+       (let [data    (get-in db [:local-storage current-chat-id])
+             bot-db  (get bot-db (or bot current-chat-id))
+             path    [(if (= :command type) :commands :responses)
+                      name
+                      :params
+                      parameter-index
+                      :suggestions]
+             args    (-> (get-in db [:chats current-chat-id :input-text])
+                         (input-model/split-command-args)
+                         (rest))
+             seq-arg (get-in db [:chats current-chat-id :seq-argument-input-text])
+             to      (get-in db [:contacts/contacts current-chat-id :address])
+             params  {:parameters {:args    args
+                                   :bot-db  bot-db
+                                   :seq-arg seq-arg}
+                      :context    (merge {:data data
+                                          :from current-account-id
+                                          :to   to}
+                                         (input-model/command-dependent-context-params current-chat-id command))}]
+         {:chat-fx/call-jail {:jail-id (or bot owner-id current-chat-id)
+                              :path    path
+                              :params  params
+                              :callback-events-creator (fn [jail-response]
+                                                         [[:received-bot-response
+                                                           {:chat-id         current-chat-id
+                                                            :command         command
+                                                            :parameter-index parameter-index}
+                                                           jail-response]])}})))))
+
+(register-handler-fx
+ ::send-message
+ [trim-v]
+ (fn [{{:keys [current-public-key current-account-id] :as db} :db} [command chat-id]]
+   (let [text (get-in db [:chats chat-id :input-text])
+         data   {:message  text
+                 :command  command
+                 :chat-id  chat-id
+                 :identity current-public-key
+                 :address  current-account-id}
+         events [[:set-chat-input-text nil chat-id]
+                 [:set-chat-input-metadata nil chat-id]
+                 [:set-chat-ui-props {:sending-in-progress? false}]]]
+     {:dispatch-n (if command
+                    (conj events [:check-commands-handlers! data])
+                    (if (str/blank? text)
+                      events
+                      (conj events [:prepare-message data])))})))
+
+(register-handler-fx
+ :proceed-command
```

Comment:
that's the point I think it would be good to have all events namespaced to know where they are coming from, and if the ns seems at odds then it might mean the event should be registered somewhere else